src/lib/nlcManager.js

```diff
@@ -133,6 +147,44 @@ NLCManager.prototype.classify = function(text){
 
 
 /**
+ * Gets data used to train the classifier with classifierId.
+ *
+ * @param  String	classifierId
+ * @return Promise  JSON			Sample result:	{"className": ["Text 1.", "Text 2"]}
+ */
+NLCManager.prototype.getClassifierData = function(classifierId){
+	logger.debug(`${TAG} Requested data used to train NLC with clasifierId=${classifierId}`);
+	return new Promise((resolve, reject) => {
+		return nlcDb.open().then((db) => {
+			return db.get(classifierId).then((doc) => {
+				csvParse(doc.trainedData, function(err, jsonData){
+					if (err){
+						logger.error(`${TAG}: Error parsing CSV data used to train NLC with clasifierId=${classifierId}`, err);
+						reject(err);
+					}
```

Comment:
The code in the 'if (err)' block doesn't do a return, so code below here needs to be wrapped in an 'else' block.